README.md

```diff
@@ -88,6 +238,10 @@ $ kubectl delete -f csi.ibm.com_v1_ibmblockcsi_cr.yaml
 $ kubectl delete -f deploy/ibm-block-csi-operator.yaml
```

Comment:
I think no need to mentioned the folder name (deploy) as the file can be downloaded to any folder..
it should be:  kubectl delete -f ibm-block-csi-operator.yaml